lore/settings.py

```diff
@@ -104,7 +106,7 @@ def get_var(name, default):
     'guardian.backends.ObjectPermissionBackend',
 )
 
-ROOT_URLCONF = 'ui.urls'
+ROOT_URLCONF = 'lore.urls'
```

Comment:
Would you rather have these listed in ui.urls, then include the rest urls from there? I am fine either way